README.md

```diff
@@ -53,26 +58,39 @@ debs:
 * python-cloud-info-provider-opennebula
 * python-cloud-info-provider
 
-#### OpenStack/ooi provider dependencies
+### From source
 
-* python-novaclient
-* python-glanceclient
-* python-keystoneauth1
+#### Using pip
 
-#### OpenNebula provider dependencies
+Source-based installation is not recommended for production usage, but is very
+handy for testing or development purpose.
+Get the source by cloning this repo and do a pip install.
 
-* python-defusedxml
+As pip will have to copy files to /etc/cloud-info-provider directory, the
+installation user should be able to write to it, so it is recommended to create
+it before using pip.
 
-### Binary packages
+```sh
+sudo mkdir /etc/cloud-info-provider
+sudo chgrp you_user /etc/cloud-info-provider
+sudo chmod g+rwx /etc/cloud-info-provider
+```
 
-Latest packages are made available at [EGI's AppDB](https://appdb.egi.eu/store/software/cloud.info.provider).
+```sh
+git clone https://github.com/EGI-Foundation/cloud-info-provider
+cd cloud-info-provider
+pip install .
+```
 
-Packages having gone through a the [EGI CMD](https://wiki.egi.eu/wiki/EGI_Cloud_Middleware_Distribution)
-[Stagged Rollout and SQA process](https://wiki.egi.eu/wiki/EGI_Cloud_Middleware_Distribution_process)
-are available in the [CMD repositories](http://repository.egi.eu/).
+##### Provider dependencies
 
-Use the appropriate repository for your distribution and install using the
-OS-specific tools.
+The OpenStack/ooi providers require the following extra libraries:
```

Comment:
Looking at `requirements.txt` it seems that other packages are needed but not listed here (for OS and ON).

Reply:
OK, you also updated `requirements.txt`